vms/avm/state/state.go

```diff
@@ -38,7 +42,7 @@ var (
 	timestampKey     = []byte{0x01}
 	lastAcceptedKey  = []byte{0x02}
```

Comment:
This isn't technically needed as part of this PR, but I would suggest changing these from bytes to plain strings (i.e `"is_initialized"`, `"timestamp"`, etc). I don't think the overhead is an issue + it makes it much easier to read in the debugger.